src/components/row/__snapshots__/__spec__.js.snap

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Row columnClasses passes the prop to its children 1`] = `
+<div
+  className="carbon-row carbon-row--gutter-medium carbon-row--columns-0"
```

Comment:
This snapshot is not testing what it is supposed to test (that columnClasses is passed down to the child column components - this example doesn't even have any child column components)